sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/FileIndex.kt

```diff
@@ -22,7 +22,7 @@ import com.squareup.sqldelight.core.SqlDelightDatabaseProperties
 import com.squareup.sqldelight.core.SqlDelightFileIndex
 import com.squareup.sqldelight.core.lang.SqlDelightFile
 import com.squareup.sqldelight.intellij.util.isAncestorOf
-import org.jetbrains.kotlin.idea.refactoring.toPsiDirectory
+import org.jetbrains.kotlin.idea.core.util.toPsiDirectory
```

Comment:
@AlecStrong These show up red in the IDE but are required for CLI builds. Something to look in to